charts/ceph-csi-cephfs/README.md

```diff
@@ -71,3 +71,70 @@ If you want to delete the namespace, use this command
+| `provisioner.attacher.enabled`             | Specifies whether attacher sidecar is enabled                                                                                                    | `true`                                             |
+| `provisioner.resizer.image.repository`     | Specifies the csi-resizer sidecar                                                                                                                | `k8s.gcr.io/sig-storage/csi-resizer`               |
+| `provisioner.resizer.image.tag`            | Specifies image tag                                                                                                                              | `v1.2.0`                                           |
+| `provisioner.resizer.image.pullPolicy`     | Specifies pull policy                                                                                                                            | `IfNotPresent`                                     |
+| `provisioner.resizer.name`                 | Specifies the name of csi-resizer sidecar                                                                                                                               | `resizer`                                          |
+| `provisioner.resizer.enabled`              | Specifies whether resizer sidecar is enabled                                                                                                     | `true`                                             |
+| `provisioner.snapshotter.image.repository` | Specifies the csi-provisioner sidecar                                                                                                            | `k8s.gcr.io/sig-storage/csi-attacher`              |
+| `provisioner.snapshotter.image.tag`        | Specifies image tag                                                                                                                              | `v4.1.1`                                           |
+| `provisioner.snapshotter.image.pullPolicy` | Specifies pull policy                                                                                                                            | `IfNotPresent`                                     |
+| `provisioner.nodeSelector`                 | Specifies the nodeSelector                                                                                                                       | `{}`                                               |
+| `provisioner.tolerations`                  | Specifies the tolerations                                                                                                                        | `{}`                                               |
+| `provisioner.affinity`                     | Specifies the affinity                                                                                                                           | `{}`                                               |
```

Comment:
mentiond about provisioner like ` Specify node selector  for provisioner deployment`